.github/workflows/codecov.yml

```diff
@@ -0,0 +1,71 @@
+# Run cargo-llvm-cov and upload to codecov.io
+
+name: Code Coverage
+
+on:
+  workflow_dispatch:
+  push:
+    # branches:
+    #   - main
+    paths:
+      - '**.rs'
+      - '.github/workflows/codecov.yml'
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: ${{ github.ref_name != 'main' }}
+
+jobs:
+  coverage:
+    name: Code Coverage
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+
+      - name: Install Rust Toolchain
+        uses: ./.github/actions/rustup
```

Comment:
Can you use 

https://github.com/rolldown/rolldown/blob/8b0a28a8237a8d0a23aa9fb1e8235292667d0f6a/.github/workflows/ci.yml#L92-L96

instead of using this hacky file 😅